assets/scss/admin/metaboxes/_metabox-award-engagement.scss

```
@@ -0,0 +1,23 @@
+.llms-award-engagement-submitbox .llms-mb-list {
+	&.student-info,
+	&.post_author_override label {
+		&:before {
+			// dashicons-admin-users.
+			font: normal 20px/1 dashicons;
+			speak: never;
+			display: inline-block;
+			margin-left: -1px;
+			padding-right: 3px;
+			vertical-align: top;
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+			content: "\f110";
+			position: relative;
+			top: -1px;
+			color: #8c8f94;
+			body:not(.admin-color-fresh) & {
+				color: currentColor; // Used when selecting a different admin color scheme from the default one.
+			}
+		}
+	}
```

```suggestion
	}
	&.post_author_override label {
		display: inline-block;
		width: auto;;
	}
```

Visually moves the "(required)" element onto the same line which I think is probably how it should always be but I haven't dug into it. This makes it look good here.

---

I was about but then I thought maybe not, and ended up choosing "not" :D